moabb/datasets/download.py

```diff
@@ -206,30 +206,39 @@ def fs_issue_request(method, url, headers, data=None, binary=False):
 
 def fs_get_file_list(article_id, version=None):
     """List all the files associated with a given article.
-
```

Comment:
Please avoid changing docstring formatting, it is done to generate automatically the doc on the website, you could revert those two line suppression.